client/state/sites/test/actions.js

```diff
@@ -215,7 +215,9 @@ describe( 'actions', () => {
 
 		test( 'should dispatch receive deleted site when request completes', async () => {
 			await deleteSite( 2916284 )( spy, getState );
-			expect( spy ).toHaveBeenCalledWith( receiveDeletedSite( 2916284 ) );
+			const expected = receiveDeletedSite( 2916284 ).toString();
```

Comment:
What is `toString()` going to return here? The function definition right?

Reply:
Yep. This is a workaround so we don't have to move actions to separate modules in this PR, as I've pointed out in https://github.com/Automattic/wp-calypso/pull/53817#discussion_r654414643.